.github/workflows/docs-push.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         run: |
           # Specify the files or directories you want to copy
           cp -r docs/* knowledge-base/developer-documentation/wallet-sdk
+          cp README.md knowledge-base/developer-documentation/wallet-sdk
+          cp -r examples/* knowledge-base/developer-documentation/wallet-sdk
```

Comment:
do you want to keep the examples in a separate folder? If you removed the `*` (i.e. `cp -r examples/ knowledge-base/developer-documentation/wallet-sdk` it should create a sub-folder for them

Reply:
I do! I changed it in the other commit